test/fixtures/README.md

```diff
@@ -0,0 +1,126 @@
+# Generate test vectors for new ZCash testnet consensus branch IDs
```

Comment:
awesome!

long-term we should have a script for this, but we can tackle that when we overhaul this lib